Cargo.toml

```diff
@@ -24,9 +24,9 @@ with_serde = ["uuid/serde", "chrono/serde", "serde"]
 regex = "1.1.0"
 lazy_static = "1.2.0"
 uuid = { version = "1.0.0" }
-chrono = { version = "0.4.6" }
+chrono = { version = "0.4.23" }
```

Comment:
semver compatible changes shouldn’t really make a difference in downstream consumers. So I guess we can just close the rustsec ticket in symbolicator.